src/wallet/wallet.cpp

```diff
@@ -1705,10 +1705,9 @@ int64_t CWallet::RescanFromTime(int64_t startTime, const WalletRescanReserver& r
  * the main chain after to the addition of any new keys you want to detect
  * transactions for.
  */
-CWallet::ScanResult CWallet::ScanForWalletTransactions(const uint256& start_block, int start_height, std::optional<int> max_height, const WalletRescanReserver& reserver, bool fUpdate)
+CWallet::ScanResult CWallet::ScanForWalletTransactions(const uint256& start_block, int start_height, std::optional<int> max_height, const WalletRescanReserver& reserver, bool fUpdate, const bool save_progress)
 {
-    using Clock = std::chrono::steady_clock;
```

Comment:
In commit "wallet: Save wallet scan progress" (86698868cdb878140c7d6a3909d6aa6f024d8cbf)

This commit doesn't compile with this line removed. The line should be removed in the next commit (which doesn't compile with the line added)

Reply:
Fixed it in https://github.com/bitcoin/bitcoin/pull/25036/commits/0ef383dd058a783bf0912c8562597823e2dbb29b. Thanks.